css/style.css

```diff
@@ -6,3 +6,228 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+body {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    margin: 0 100px 40px 100px;
+}
+
+main {
+    margin-bottom: 50px;
+
+}
+
+  /* Navigation Styles */
+
+ul {
+      list-style: none;
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      margin-bottom: 50px;
+}
+
+ul li a {
+      color: rgb(104, 89, 89);
+      text-decoration: none;
+      font-weight: 600;
+      padding: 10px;
+}
+
+ul li a:hover {
+      color: black;
+}
+
+.border-bottom {
+    border-bottom: 1px solid black;
+}
+
+ /* Hero Styles */
+
+.left-container {
+     display: flex;
+     justify-content: flex-end;
+     align-items: center;
+     flex-direction: column;
+     background-image: url("https://images.pexels.com/photos/1879877/pexels-photo-1879877.jpeg?auto=compress&cs=tinysrgb&w=600");
+     background-position: cover;
+     background-repeat: no-repeat;
+     color: whitesmoke;
+     height: 700px;
+     width: 50%;
+     padding-bottom: 50px;
+}
+
+ .h3 {
+     text-decoration: underline;
+}
+
+.h1 {
+     font-size: 30px;
+ }
+
+.date {
+     font-weight: bolder;
+}
+
+/* General Styles - Top Container*/
+
+.top-container {
+    padding: 5px;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    margin-bottom: 0;
+    
+}
+ 
+.logo {
+     display: flex;
+     flex-direction: row;
+     font-size: 80px;
+     font-weight: bold;
+     justify-content: center;
+     padding-right: 68px;
+     text-decoration: underline;
+}
+
+.top-right {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+ }
+
+ .top-right-p1 {
+     padding: 5px 0 0 100px;
+     font-weight: 600;
+}
+
+.top-right-p2 {
+    padding: 5px 0 0 10px;
+    font-weight: 600;
+}
+
+.hburger-img {
+    width: 60px;
+    height: 60px;
+}
+
+h4 {
+    color: rgb(88, 85, 85);
+}
+
+/* General Styles - Main container and aside container */
+.main-container {
+    display: flex;
+    flex-direction: row;
+    height: 800px;
+    margin: 50px 0 10px 0;
+}
+
+.aside-container {
+    float: right;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;   
+    padding-left: 100px;
+}
+
+.aside-border {
+    border: 1px solid rgb(180, 175, 175);
+    margin-bottom: 40px;
+    padding: 15px;
+    font-weight: bolder;  
+}
```

Comment:
You could add this code to .aside-container preventing the need for an extra element to add a border